web-server/v0.4/src/utils/parse.js

```diff
@@ -290,3 +283,38 @@ export const parseIterationData = results => {
     ports: ports,
   };
 };
+
+export const filterIterationData = (responseData, selectedConfig) => {
+  const responseDataCopy = [];
+  for (var response in responseData) {
+    responseDataCopy[response] = [];
+    responseDataCopy[response].columns = cloneDeep(responseData[response].columns);
+    responseDataCopy[response].iterations = cloneDeep(responseData[response].iterations);
+    responseDataCopy[response].resultName = cloneDeep(responseData[response].resultName);
+  }
+
+  for (var response in responseDataCopy) {
+    const responseIterations = responseDataCopy[response].iterations;
+    let selectedConfigLength = Object.keys(selectedConfig).length;
+    if (selectedConfigLength > 0) {
+      const filteredResponseData = [];
+      for (let iteration in responseIterations) {
+        let found = [];
+        for (let config in selectedConfig) {
+          if (
+            (selectedConfig[config] !== undefined) &
+            (selectedConfig[config] == responseIterations[iteration][config])
+          ) {
+            found.push(true);
```

Comment:
Since we are only using `found.length` below which is a count, should we just use a counter instead of an array here?

Reply:
Yes! Using a counter here will improve performance and save memory being allocated.